gradle/libs.versions.toml

```diff
@@ -0,0 +1,21 @@
+[versions]
+
```

Comment:
Do we need these empty lines after the section? The examples on https://toml.io/en/ don't have them

Reply:
Empty lines are removed